### DISCUSSION TITLE
Multisite prices and currencies

### DISCUSSION THREAD

**Discussion by sattyframework** [duncanmcclean/simple-commerce#941] [Help]
Hey there, 
I am currently working on a project which is using multi site. Currently it has two sites, UK being the default and US. I have set up my sites in my config files. 
I have my variants set up for products and I can see a price field. I can see that the currency changes when going between the different sites, however the price stays the same. 
My question is, is it possible to have different prices for the different sites?
Also I have created a duplicate product in the CMS for US site and have made the product_variant field localizable, however the price fields within are set to read only and I cant figure out how to edit them. 

Any help is appreciated, thanks.

**Comment by duncanmcclean** [comment]
You should be able to mark the "Price" field as localizable so it can be different on each site?

**Reply by sattyframework** [reply]
I actually cant see the price field when I click product_variant in the blueprint (not sure if its a hidden field). 
I can set localizable on the parent field like below but any fields inside do not have the globe icon and show up as 'read only' in other sites

![Screenshot 2023-11-28 at 10 58 27](https://github.com/duncanmcclean/simple-commerce/assets/109515230/d9a26b51-63f9-4bf5-a58d-9fad76c64ea1)

**Reply by duncanmcclean** [reply]
Ah, sorry, didn't see that you mentioned product variants. I think some work needs done to make variants localizable. I've opened a bug report here: https://github.com/duncanmcclean/simple-commerce/issues/942
